packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts

```diff
@@ -30,33 +30,24 @@ context('Access to pagelist', () => {
     cy.getByTestid('page-duplicate-modal').should('be.visible').within(() => {
       cy.get('.rbt-input-main').type('-duplicate', {force: true})
     }).screenshot(`${ssPrefix}4-input-duplicated-page-name`);
-    cy.getByTestid('page-duplicate-modal').should('be.visible').within(() => {
-      cy.get('.modal-footer > button').click();
-    });
     cy.get('body').type('{esc}');
-    cy.getByTestid('pageListButton').click({force: true});
-    cy.getByTestid('page-accessories-modal').parent().should('have.class','show').within(() => {
-      cy.get('.list-group-item').eq(0).within(() => {
-        cy.screenshot(`${ssPrefix}5-duplicated-page`);
-      });
-    });
```

Comment:
what is the reason for removing these?

Reply:
Because previously duplicate pages made the order and contents of the page list/timeline change, so I think skipping it will reduce page list changes. I'll revert it while minimizing changes to the page list/timeline